.travis.yml

```diff
@@ -39,6 +39,15 @@ env:
   - LAYER=xdebug PHP="72 73 74"
   - LAYER=yaml PHP="72 73 74"
 
+cache:
+  directories:
+  - docker_images
+
+before_cache:
+  # Save docker images
+  - mkdir -p docker_images
+  - docker save -o docker_images/images.tar $(docker images -a -q)
```

Comment:
```suggestion
  - docker save -o docker_images/images.tar $(docker images -a -q)
  - ls -l docker_images/images.tar
```
Is there anything at all?

![](https://user-images.githubusercontent.com/139488/97518621-f5108e80-19da-11eb-9f70-230950a59dcf.png)